toolsWebsite/tools/views.py

```diff
@@ -71,13 +71,64 @@ def handle_uploaded_file(f):
     with open(f"tools/static/images/{f}", 'wb+') as destination:
         for chunk in f.chunks():
             destination.write(chunk)
+
+def checkout(request):
+    # We can do all sorts of stuff with an html template with the cart info
+    # that will be stored in our cart models.  I.e. tell the customer when it will be
+    # due/ do a confirmation page etc.
+
+    for cart in ShoppingCart.objects.all():
+        due = DueDates()
+        due.toolCategory = cart.toolCategory
+        due.quantity = cart.quantity
+        due.date_bought = datetime.datetime(2019, randint(1, 12), randint(1, 28))
+        due.date_due = datetime.datetime(2020, randint(1, 12), randint(1, 28))
+        # due.date_bought = datetime.datetime.now()
+        # due.date_due = datetime.datetime.now()
+        due.buyer = "Connor"
```

Comment:
I think we should put these fields as forms before completing the checkout

Reply:
So `date_bought` should be `datetime.now()` and `date_due` should be a datepicker where we can choose which date the tool is supposed to be returned, and `buyer` should be a textbox where we can enter the customer's name.

Reply:
We want to do a checkout verification page or checkout detail page, or whatever we want to call it then?  I'm going to get it working without that first and then let's make that a requirement for the next sprint so we can have a meeting on how exactly we want to style it.